### ISSUE TITLE
Add documentation on which versions of Django are supported

### ISSUE THREAD

**Issue by jchubber** [wooey/Wooey#266]
It would be helpful to understand which versions of Django have been tested and/or are supported. For example, does Wooey work with Django 2.0 or higher? If not, it would be helpful for myself and others who might want to use Wooey on Django 2+ to be able to contribute pull requests.

**Comment by Chris7** [comment]
Good idea. The current place to look at this is in the builds: https://travis-ci.org/wooey/Wooey. I would love any PRs that help us get on Django 2 as well.

**Comment by jchubber** [comment]
Cool. For the documentation, would you like me to submit a PR to add to `Readme.md` in the root folder `/`, or would you prefer that I submit a PR to add somewhere in `/docs/`? Or both?

**Comment by Chris7** [comment]
I'm about to leave for a few weeks on vacation, but putting it in docs would be best for me. I have some plans for a refactor of hte README.md
